pages/hackathon-grant.js

```diff
@@ -0,0 +1,195 @@
+                    as="a"
+                    sx={{
+                      width: 72,
+                      height: 72,
+                      backgroundImage: "url('/hackathon-grant/first.png')",
+                      backgroundColor: '#231F20',
+                      backgroundSize: '60px, cover',
+                      backgroundPosition: 'center center',
+                      backgroundRepeat: 'no-repeat',
+                      borderRadius: 8
+                    }}
+                  ></Box>
+                </NextLink>
+              </Flex>
+              A <Sparkles>$500</Sparkles> grant for your in-person hackathon.
+            </Heading>
+            <Button variant="ctaLg" as="a" href="#apply" sx={{ mt: 2 }}>
+              Coming soon
+            </Button>
+          </Box>
+        </Box>
+        <Container sx={{ py: 5 }}>
+          <Box sx={{ fontSize: '27px', textAlign: 'center', mb: 4 }}>
+            Hack Club is partnering with{' '}
+            <NextLink
+              href="https://www.firstinspires.org"
+              target="_blank"
+              passHref
+            >
+              <Link>
+                <i>FIRST</i>
+              </Link>
+            </NextLink>{' '}
+            to provide $500 grants to in-person high-school hackathons happening
+            this semester.
+          </Box>
+          <Text
+            as="h1"
+            variant="heading"
+            mb={3}
+            sx={{ fontSize: 50, textAlign: 'center' }}
+          >
+            Requirements
+          </Text>
+
+          <Grid columns={2} mb={5}>
+            <Requirement title="Proof of venue">
+              Provide a scan of a contract, or an{' '}
+              <Link href="https://www.investopedia.com/terms/m/mou.asp">
+                MOU (memorandum of understanding)
+              </Link>
+              .
+            </Requirement>
+            <Requirement title="Event must be fully in-person">
+              Virtual and hybrid events aren't eligible.
+            </Requirement>
+            <Requirement
+              title={
+                <>
+                  Run on{' '}
+                  <NextLink href="/bank" passHref>
+                    <Link>Hack Club Bank</Link>
+                  </NextLink>{' '}
+                  with Transparency Mode enabled
+                </>
+              }
+            >
+              You'll receive your grant through Hack Club Bank, and spend it
+              transparently through{' '}
+              <Link
+                href="https://changelog.bank.hackclub.com/transparent-finances-(optional-feature)-151427"
+                target="_blank"
+              >
+                Transparency Mode
+              </Link>
+              .
+            </Requirement>
+            <Requirement title="Handmade website">
+              Your event's website should be coded by you— not a site builder
+              like Wix or Google Sites. Need help? Check out our{' '}
```

Comment:
Would it be worth noting that they should also have an event-specific domain and that they can get one for free when they run on HCB?